src/SFA.DAS.ApprenticeFeedback.Database/StoredProcedures/ImportIntoFeedbackTargetVariant_FromFeedbackTargetVariant_Staging.sql

```diff
@@ -0,0 +1,17 @@
+﻿CREATE PROCEDURE [dbo].[ImportIntoFeedbackTargetVariant_FromFeedbackTargetVariant_Staging]
+AS
+BEGIN
+    MERGE INTO [dbo].[FeedbackTargetVariant] AS TARGET
+    USING [dbo].[FeedbackTargetVariant_Staging] AS SOURCE
+    ON TARGET.ApprenticeshipId = SOURCE.ApprenticeshipId
+    WHEN NOT MATCHED BY TARGET THEN
+        INSERT (ApprenticeshipId, Variant)
+        VALUES (SOURCE.ApprenticeshipId, SOURCE.Variant)
+    WHEN MATCHED THEN
+        UPDATE SET TARGET.Variant = SOURCE.Variant;
+
+    DELETE FROM [dbo].[FeedbackTargetVariant]
```

Comment:
WHEN NOT MATCHED BY SOURCE THEN
        DELETE; 

You can do the delete as part of the merge

Reply:
updated